test/DynamoCoreWpfTests/AnnotationViewModelTests.cs

```diff
@@ -51,6 +51,15 @@ public void CreateGroupAroundNodes()
 
         [Test]
         [Category("DynamoUI")]
+        public void CanCreateGroupIfNoModelsAreSelectedInTheCanvas()
```

Comment:
In general the group tests need to have names that reflect whether the test is POS or NEG